vms/platformvm/txs/executor/camino_tx_executor.go

```diff
@@ -1477,27 +1479,61 @@ func addCreds(tx *txs.Tx, creds []verify.Verifiable) {
 }
 
 func (e *CaminoStandardTxExecutor) AddressStateTx(tx *txs.AddressStateTx) error {
-	if err := e.Tx.SyntacticVerify(e.Ctx); err != nil {
+	var err error
```

Comment:
any specific reason for this (aside from avoiding to declare new err var each time with `:=`)

Reply:
Yes: https://github.com/chain4travel/caminogo/pull/262/files#diff-dc16653a79c2742b3d1f972d4b41788ebd70143553cf00835c3e5e637f4de2aaR1508